### ISSUE TITLE
failed with shopify

### ISSUE THREAD

**Issue by molitona** [Maltemo/favicon-to-murmurhash#1]
the tool got incorrect favicon link from shopify.com
-> https://www.shopify.com/favicon.ico
correct one is https://cdn.shopify.com/static/shopify-favicon.png

**Comment by Maltemo** [comment]
The favicon is correctly detected. 
If you open your web console, you will see this warning with this message: `CORS blocking favicon access of https://cdn.shopify.com/static/shopify-favicon.png. Sending it to background`
The problem is that as we fetch the image from "https://www.shopify.com", but the image is hosted on the subdomain "cdn.shopify.com". Consequently, CORS are blocking the request because it's not on the same site.

For the moment I didn't found any viable solution to access those favicons blocked by CORS without infringing on the privacy of users of the extension (going through a proxy to fetch the favicon would enable the owner of the proxy to monitor the websites you are visiting).

If you have some suggestions, don't hesitate.

**Comment by Maltemo** [comment]
A new issue will be created soon to resolve this problem, closing this issue.
